eng/test-build-correctness.ps1

```diff
@@ -33,6 +33,19 @@ try {
   . (Join-Path $PSScriptRoot "build-utils.ps1")
   Push-Location $RepoRoot
 
+  # Verify no PROTOTYPE marker left in master
+  if ($env:SYSTEM_PULLREQUEST_TARGETBRANCH -eq "master")
+  {
+    Write-Host "Checking no PROTOTYPE markers in compiler source"
+    Get-ChildItem -Path src/Compilers/*.cs,src/Compilers/*.xml -Recurse | Select-String -Pattern 'PROTOTYPE' -CaseSensitive -SimpleMatch -OutVariable prototypes | Out-Null
```

Comment:
Why use `OutVariable` and not just assign the result?

Reply:
this is my first Powershell change, so it may not be idiomatic ;-) Are you saying `$prototypes = ... | ...` would work too?

Reply:
I think it should. I'll defer to @jaredpar though.

Reply:
Looks like that works